scripts/swarmctl.py

```
@@ -12,7 +12,18 @@
+REPO_SRC_DIR = SCRIPT_DIR.parent / "repos/packages/agent-os/src"
+SRC_CANDIDATES = []
+if os.getenv("AGENT_OS_SRC"):
+    SRC_CANDIDATES.append(Path(str(os.getenv("AGENT_OS_SRC"))))
+SRC_CANDIDATES.append(REPO_SRC_DIR)
+SRC_CANDIDATES.append(Path("/Users/user/antigravity-core/repos/packages/agent-os/src"))
+
+SRC_DIR = next((p for p in SRC_CANDIDATES if p.exists()), None)
+if SRC_DIR is None:
+    raise SystemExit(
+        "agent-os src not found. Set AGENT_OS_SRC or ensure repos/packages/agent-os/src exists."
+    )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)